testsuite/helpers/TsCmp.cc

```diff
@@ -41,6 +55,128 @@ namespace storage
     }
 
 
+    /**
+     * This function adjusts the sids which is needed when devices are the
+     * detected in a differnet order than expected. Block devices use the name
+     * for identification, most others a uuid or a path. Some types are not
+     * handled at all, e.g. Nfs.
+     *
+     * The function makes assumptions that break in the general case and does
+     * no error checking. It can even ruin the devicegraph.
+     *
+     * Only enable it when you know what you are doing!
+     */
+    void
+    TsCmpDevicegraph::adjust_sids(const Devicegraph& lhs, Devicegraph& rhs) const
```

Comment:
Because sometimes it is needed: When the probing is modifies and the same objects are detected but in a different order. In that case the sids have to be adjusted. This is the case with the modifications I'm planning for multipath.